tests/unit/rpc_test.cc

```diff
@@ -36,6 +36,7 @@
 #include <seastar/util/log.hh>
 #include <seastar/util/closeable.hh>
 #include <seastar/util/noncopyable_function.hh>
+#include <seastar/util/later.hh>
```

Comment:
Why this?

Reply:
Those are remains of a first attempt at unit test, I will remove it.

Reply:
removed